### PR TITLE
feat: allow users to be invited

### PR DESCRIPTION
**Pull request by amorriscode**
### tl;dr

- [x] Adds list of users to the `/settings` page
- [x] Adds the ability to create/delete invite links from the `/settings` page
- [x] Allows users to sign up using invite links

### CODE REVIEW

File: src/pages/onboard.tsx [hunk 1]
```diff
@@ -30,18 +32,14 @@ const Services: Component = () => {
...
-          your incidents.
-        </p>
-      </aside>
-
-      <main class="bg-zinc-800 border border-zinc-700 rounded p-8 shadow shadow-zinc-900/50">
-        <OnboardForm />
-      </main>
+      <Switch>
+        <Match when={inviteCode}>
+          <OnboardFormWithoutOrg inviteCode={inviteCode || ''} />
```
Comment (alicelovescake) [inline src/pages/onboard.tsx:37]: do you need the || if you show when inviteCode?

File: src/components/forms/OnboardFormWithOrg.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+            })
+
+            await createOrganization({
+              input: {
+                name: organizationName,
+              },
+            })
+
+            if (createUserResult().error || createOrganizationResult().error) {
+              alert('Something went wrong, please try again.')
```
Comment (alicelovescake) [inline src/components/forms/OnboardFormWithOrg.tsx:76]: We have the Error Alert component that you can use

File: src/components/forms/OnboardFormWithoutOrg.tsx [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+type Props = {
+  inviteCode: string
+}
+
+const OnboardFormWithoutOrg: Component<Props> = ({ inviteCode }) => {
+  const [{ user }] = useContext(AuthContext)
+  const [inviteResult] = createQuery({
+    query: GET_INVITE,
+    variables: { code: inviteCode },
+  })
```
Comment (alicelovescake) [inline src/components/forms/OnboardFormWithoutOrg.tsx:40]: Should we have an expiring on these invite codes? i.e you can check the date.now compared to when it was created, if it is over 30 days, they need a new code. In cases where ppl leave and stuff. But might be overkill

Comment (amorriscode) [reply src/components/forms/OnboardFormWithoutOrg.tsx:40]: Just posting for visibility but we've decided this is a stretch feature.

**Pull request merged**